app/models/journey.rb

```diff
@@ -46,6 +46,15 @@ def keywords_search
     join_search([to_search_str(keywords), steps_search(:keywords)])
   end
 
+  def reposition
+    steps.each_with_index do |step, index|
+      position_by_index = (index + 1) * 2
+      if step.position != position_by_index
+        return false unless step.update(position: position_by_index)
```

Comment:
Tejto logike asi uplne nerozumiem. pusta sa update ked nesedi position s tym co je v db za zaroven ked sa to nepodari updatnut tak rovno koncime a skaceme von?

Reply:
Ano, skonci hned po prvom neprejdenom update stepu. Nechcel som sa pokusat menit dalsie stepy, aby situacia (existujuce poradie) zostalo co najmenej zmenene (a uzivatel sa mohol pokusit to spustit este raz). Ale ked nad tym teraz uvazujem, tak ak uz sa zmenili nejake stepy predtym, tak to poradie sa uz aj tak prehodi. A asi by teda bolo lepsie vyskusat to aj pre ostatne.